readme.md

```diff
@@ -22,67 +22,76 @@ Hello, Node!
 
 ### Instructions & Guiding Questions
 
-- [ ] Begin by renaming `data/pets.sample.json` to `pets.json`
+- [x] Begin by renaming `data/pets.sample.json` to `pets.json`
 
 * **Question:** In this lesson we will be modifying the contents of the `pets.json` file. Why do you think that file appears inside of the `.gitignore` file?
 
 * **Your Answer:**
-
+It's a file that we will be modified pretty often.
 ---
 
-- [ ] Take a look at the NodeJS documentation for the [fs module](https://nodejs.org/api/fs.html). When looking at documentation like this it can be overwhelming to start but you likely know more than you think. Take a moment to find a few concepts you understand.
+- [x] Take a look at the NodeJS documentation for the [fs module](https://nodejs.org/api/fs.html). When looking at documentation like this it can be overwhelming to start but you likely know more than you think. Take a moment to find a few concepts you understand.
 
 ---
 
-- [ ] One of the most common patterns we'll come across as web developers is the concept of [CRUD](https://www.codecademy.com/articles/what-is-crud). Take a moment to define what CRUD represents.
+- [x] One of the most common patterns we'll come across as web developers is the concept of [CRUD](https://www.codecademy.com/articles/what-is-crud). Take a moment to define what CRUD represents.
 
 * **Question:** Imagine you have a file called `classmates.txt`. For each part of CRUD, describe how the action would interact with the file.
 
 * **Your Answer:**
+Create - create a new classmate object
+Read - reads the classmate list
+Update - updates a classmate object
+Delete - deletes classmate object
 
 ---
 
-- [ ] Consider the above and then look back through the [fs module](https://nodejs.org/api/fs.html) documentation.
+- [x] Consider the above and then look back through the [fs module](https://nodejs.org/api/fs.html) documentation.
 
 * **Question:** What methods represent each CRUD action?
 
 * **Your Answer:**
 
+C - `fs.writeFile`
+R - `fs.readFile`
+U - `fs.appendFile`
+D - `fs.delete`
```

Comment:
I don't believe there is an `fs.delete()`

Reply:
Right it's not, not sure how I got that 🤔 Let me update.